doc/source/flexibility/compressed_columns.rst

```diff
@@ -0,0 +1,213 @@
+.. _compressed_columns:
+
+====================================
+Compressed columns with dictionaries
+====================================
+
+In :rn:`5.6.33-79.0` |Percona Server| has been extended with a new per-column
+compression feature. It is a data type modifier, independent from user-level
+SQL and |InnoDB| data compression, that causes the data stored in the column to
+be compressed on writing to storage and decompressed on read. For all other
+purposes the data type is identical to the one without modifier, i.e. no new
+data types are created.
+
+This feature provides:
+
+* a better compression ratio for text data which consist of a large number of
+  predefined words (e.g. JSON or XML) using compression methods with static
+  dictionaries,
+* in contrast to |InnoDB| row compression method it provides a way to select
+  which columns in the table must be compressed.
+
```

Comment:
Please, mention that currently we use `zlib` for column compression. Other algorithms may be added in future versions.